aggregate_temporal_frequency.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "aggregate_temporal_frequency",
```

Comment:
I'd suggest `aggregate_temporal_period`.

Reply:
This originates from `climatological_normal`. First we had two parameter there `period` and `climatology_period`, but it felt confusing as there's an "unnamed" period together with climatology_period so we tried to avoid that and the best solution we found was frequency. As these processes are often used together, we decided to use frequency in all to avoid confusion. I guess we could revert to period if that feels better overall. Maybe the two period parameters are not as confusing?

Reply:
Daily is still a period from 0-24h, but is for sure less often used in EO than e.g. monthly or seasonal for higher level analysis, I think. I am fine with frequency, because I am often using Pandas (e.g., [Pandas Date Range](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.date_range.html)) and there frequency is also a common term.

Reply:
Daily was an example, could also be yearly. What I meant is, it not just a single exactly specified period like 2010-2011, but here it is describing how something is split into multiple exact periods depending on the input data: 2010-2011, 2011-2012, 2012-2013, ...

Reply:
Renamed frequency to period throughout the PR. Also, it's now day instead of daily, season instead of seasons etc.